src/test/java/com/googleinterns/zoomtube/servlets/AuthenticationServletTest.java

```diff
@@ -29,26 +29,28 @@
   private AuthenticationServlet servlet;
   @Mock private HttpServletRequest request;
   @Mock private HttpServletResponse response;
```

Comment:
nit: It is good practice to have mocks at the beginning of tests.

Reply:
Moved